business/namespaces.go

```diff
@@ -65,10 +65,6 @@ func NewNamespaceService(
 	}
 }
 
-func (in *NamespaceService) clusterIsOpenShift(cluster string) bool {
```

Comment:
I think this https://github.com/kiali/kiali/pull/8663/files#diff-fefabbef0775928d76cdf5b290d8afa671cb596f0159b13b0b61d03e69a7058bR60 is not used anymore, right?

Reply:
yes right, this local function was used only in one place